text/0000-base-image-label.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Add base image SHA to stack metadata
+- Start Date: 2020-10-23)
+- Author(s): dumez-k
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Building off of this earlier [RFC](https://github.com/buildpacks/rfcs/blob/main/text/0050-stack-metadata.md) outlining stack metadata,
+we would like to add a label to expose the stack base image SHA in the metadata of all stack-cnb images.
+
+# Motivation
+[motivation]: #motivation
+
+- Allows stack users and authors to easily see which base-image that each stack cnb image is based off of.
+
+# What it is
+[what-it-is]: #what-it-is
+
+This provides a high level overview of the feature.
+
+- Define the target persona: stack user, stack author
+
+The label can be **optionally** added by stack maintainers to stack image:
+ 
+`io.buildpacks.stack.base_image:` SHA of stack base image
```

Comment:
which `SHA`? Is this the [image ID](https://github.com/opencontainers/image-spec/blob/e562b04403929d582d449ae5386ff79dd7961a11/config.md#imageid) or the manifest digest?

Reply:
image ID as defined [here](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier) but I think this is the same concept as what you're referring to